### DISCUSSION TITLE
为什么我安装不了forge

### DISCUSSION THREAD

**Discussion by 114514kobe** [Meloong-Git/PCL#4503] [提问] [answered]
游戏安装失败
→ 下载 Forge 40.2.10失败
→ 安装 Mod 加载器（方式 A）失败
→ 安装新 Forge 版本失败
→ 运行 Forge 安装器失败
→ Forge 安装器出错，日志结束部分为：
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at com.bangbang93.ForgeInstaller.main(ForgeInstaller.java:68)
   於 PCL.ModDownloadLib._Closure$__37-0._Lambda$__7(LoaderTask`2 Task)
   於 PCL.ModLoader.LoaderTask`2._Closure$__12-0._Lambda$__0()
   於 PCL.ModDownloadLib._Closure$__37-0._Lambda$__7(LoaderTask`2 Task)
   於 PCL.ModDownloadLib.ForgeInjector(String Target, LoaderTask`2 Task, String McFolder, Boolean UseJavaWrapper)
   於 PCL.ModDownloadLib._Closure$__37-0._Lambda$__7(LoaderTask`2 Task)
他给我复制的是这个求解答

**Comment by SALTWOOD** [comment]
有完整一些的日志吗？

**Reply by 114514kobe** [reply]
我不清楚她在哪里可以找到完整的日志

**Reply by Ad-closeNN** [reply]
PCL\log1.txt
记得等待 Uploading... 结束后再点击 comment

**Comment by Ad-closeNN** [comment]
~~您就是just kidding那位吗~~
- 有点像 #4476

**Reply by 114514kobe** [reply]
[Uploading Log1.txt…]()

**Reply by Ad-closeNN** [reply]
> Uploading Log1.txt…

@114514kobe 等待 Uploading... 结束后再点击 comment

**Reply by SALTWOOD** [reply]
> Uploading Log1.txt…

666
**记得等待 Uploading... 结束后再点击 comment**

**Reply by 114514kobe** [reply]
[Log1.txt](https://github.com/user-attachments/files/16622601/Log1.txt)

**Reply by 114514kobe** [reply]
好了

**Reply by Ad-closeNN** [reply]
2.6.8(332)??这么远古的版本建议先升级你的PCL版本
打不开爱发电试试
[afdian.com](https://afdian.com)
[ifdian.net](https://ifdian.net)

**Reply by SALTWOOD** [reply, marked as answer]
2.6.8……先升级再试试？

**Reply by 114514kobe** [reply]
成功了谢谢各位

**Reply by 114514kobe** [reply]
是我忘了更新

**Comment by Pigeon0v0** [comment]
这里有一个小技巧

如果是旧版本启动器，会显示「安装 Mod 加载器」

但如果是更新了 NeoForge 之后的版本，会显示「安装 Forge」
